pkg/pillar/build.yml

```diff
@@ -14,6 +14,9 @@ config:
     - /config:/config
     - /:/hostfs
     - /persist:/persist:rshared,rbind
+    - /usr/bin/containerd:/usr/bin/containerd
+    - /usr/bin/containerd-shim:/usr/bin/containerd-shim
+    - /usr/bin/containerd-shim-runc-v2:/usr/bin/containerd-shim-runc-v2
```

Comment:
We will need `ctr` as well, and `runc` too.

Reply:
I use system containerd to run domains, so no needed

Reply:
We still need the binaries.